skype/skype-ps/skype/Set-CsTeamsFeedbackPolicy.md

```diff
@@ -0,0 +1,179 @@
+---
+external help file: Microsoft.Rtc.Management.dll-help.xml
+online version: https://docs.microsoft.com/powershell/module/skype/set-csteamsfeedbackpolicy
+applicable: Skype for Business Online
+title: Set-CsTeamsFeedbackPolicy
+schema: 2.0.0
+manager: bulenteg
+ms.author: tomkau
+author: tomkau
+ms.reviewer:
+---
+
+# Set-CsTeamsFeedbackPolicy
+
+## SYNOPSIS
+
+Use this cmdlet to modify a Teams feedback policy.
+
+## SYNTAX
+
+```
+Set-CsTeamsFeedbackPolicy [-WhatIf] [-Confirm] [[-Identity] <Object>] [-Tenant <Object>]
+ [-ReceiveSurveysMode <String>] [-UserInitiatedMode <String>] [-Force] [-Instance <Object>]
+```
+
+## DESCRIPTION
+Modifies a Teams feedback policy.
+
+## EXAMPLES
+
+### Example 1
+```powershell
+PS C:\> Set-CsTeamsFeedbackPolicy -identity "New Hire Feedback Policy" -userInitiatedMode enabled -receiveSurveysMode disabled
+```
+
+In this example, the policy "New Hire Feedback Policy" is modified, sets the userInitiatedMode parameter to enabled and the receiveSurveysMode parameter to disabled.
+
+## PARAMETERS
+
+### -Confirm
+Prompts you for confirmation before running the cmdlet.
+
+```yaml
+Type: SwitchParameter
+Parameter Sets: (All)
+Aliases: cf
+
+Required: False
+Position: Named
+Default value: None
+Accept pipeline input: False
+Accept wildcard characters: False
+```
+
+### -Force
+Suppresses any confirmation prompts that would otherwise be displayed before making changes and suppresses the display of any non-fatal error message that might arise when running the command.
+
+```yaml
+Type: SwitchParameter
+Parameter Sets: (All)
+Aliases:
+
+Required: False
+Position: Named
+Default value: None
+Accept pipeline input: False
+Accept wildcard characters: False
+```
+
+### -Identity
+The unique identifier of the policy.
+
+```yaml
+Type: Object
+Parameter Sets: (All)
+Aliases:
+
+Required: False
+Position: 1
+Default value: None
+Accept pipeline input: False
+Accept wildcard characters: False
+```
+
+### -Instance
+Internal Microsoft use.
+
+```yaml
+Type: Object
+Parameter Sets: (All)
+Aliases:
+
+Required: False
+Position: Named
+Default value: None
+Accept pipeline input: False
+Accept wildcard characters: False
+```
+
+### -ReceiveSurveysMode
+Set the receiveSurveysMode parameter to enabled to allow users who are assigned the policy to receive the survey.
+
+Possible values:
+ - Enabled
+ - Disabled
+
+```yaml
+Type: String
+Parameter Sets: (All)
+Aliases:
+
+Required: False
+Position: Named
+Default value: Enabled
```

Comment:
```suggestion
Default value: None
```

I prefer no default value for Set- but if you disagree, I think it should be the same as New-.